src/core_seaice/shared/mpas_seaice_advection_incremental_remap.F

```diff
@@ -4596,10 +4558,9 @@ subroutine compute_gradient_3d(&
           !TODO - Rotate the gradient vector if on a plane?
 
           if (config_rotate_cartesian_grid .and. on_a_sphere) then
-             globalGradRotate(:,:,1) = -globalGrad(:,:,3)   ! xR = -z
-             globalGradRotate(:,:,2) =  globalGrad(:,:,2)   ! yR = y
-             globalGradRotate(:,:,3) =  globalGrad(:,:,1)   ! zR = x
-             globalGrad(:,:,:) = globalGradRotate(:,:,:)
+             normalGrad(:,:,1) =  globalGrad(:,:,1)   ! temp
```

Comment:
Could you add a comment that makes is really clear normalGrad is being reused as a temporary variable here?